packages/web/src/scss/tools/__tests__/_responsive-properties.test.scss

```diff
@@ -0,0 +1,122 @@
+@use 'true' as test;
+@use '../responsive-properties';
+
+@include test.describe('responsive properties functions and mixins') {
+    @include test.it('should return correct property with custom property cascade') {
+        @include test.assert() {
+            @include test.output() {
+                .test {
+                    @include responsive-properties.create-cascade(
+                        $property: 'gap',
+                        $input-custom-property-base-name: '--test-gap',
+                        $breakpoints: (
```

Comment:
we can load breakpoints from tokens too... We do it elsewhere... But I understand this might be more stable.

Reply:
I am declaring the breakpoints manually on purpose so the code is (in my belief) more readable.